bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/upgrade.yaml

```diff
@@ -281,9 +299,23 @@
     kubectl wait --for=condition=complete --timeout=300s --namespace {{ namespace }} job/airflow-update-check-first-run
 
 - name: "(5/6) Upgrade Astronomer: Helm upgrade - Update Airflow configurations. Watch for Airflow charts to upgrade version with 'helm list --all-namespaces', watch for airflow pods to start back up."
-  shell: |
-    helm upgrade --namespace {{ namespace }} --reset-values -f {{ astro_save_dir.path }}/helm-user-values.json --set astronomer.houston.upgradeDeployments.enabled=true --set astronomer.airflowChartVersion={{ upgrade_to_version_airflow }} --version={{ upgrade_to_version }} {{ release_name }} astronomer/astronomer
+  shell: >
+    helm upgrade 
+    --namespace {{ namespace }} 
+    --reset-values 
+    -f {{ astro_save_dir.path }}/helm-user-values.json 
+    --set astronomer.houston.upgradeDeployments.enabled=true 
+    --set astronomer.commander.upgradeTimeout={{ COMMANDER_UPGRADE_TIMEOUT }} 
+    --version={{ upgrade_to_version }} {{ release_name }} astronomer/astronomer
+  when: not disable_upgrade_timeout
 
 - name: "(6/6) Upgrade Astronomer: Helm upgrade - Turn off airflow auto upgrade configuration. Perfunctory step, nothing to look for."
-  shell: |
-    helm upgrade --namespace {{ namespace }} --reset-values -f {{ astro_save_dir.path }}/helm-user-values.json --no-hooks --set astronomer.houston.upgradeDeployments.enabled=false --version={{ upgrade_to_version }} {{ release_name }} astronomer/astronomer
+  shell: >
+    helm upgrade 
+    --namespace {{ namespace }} 
+    --reset-values 
+    -f {{ astro_save_dir.path }}/helm-user-values.json 
+    --no-hooks --set astronomer.houston.upgradeDeployments.enabled=false 
+    --set astronomer.commander.upgradeTimeout=300 
+    --version={{ upgrade_to_version }} {{ release_name }} astronomer/astronomer
+  when: not disable_upgrade_timeout
```

Comment:
Do we need this here .. seems like astronomer.houston.upgradeDeployments.enabled is hardcoded to false here.

Reply:
@vishwas-astro this could run without harm but isn't needed to run to finish an upgrade. That step turns off upgrade deployments in the values after we set it.

Reply:
to make it more clear, we could put steps 5 and 6 in a `block:` 🤷‍♂️